ceph-releases/luminous/ubuntu/16.04/daemon/disk_introspection.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/bash
+
+set -e
+
+DISK_FILE=$(hostname -f)-disks
```

Comment:
We should query the Kubernetes API to determine the hostname of the node we're running on.